### PR TITLE
added self to Documentation list

### PR DESCRIPTION
**Pull request by cinemassacres**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable

# Description

### CODE REVIEW

File: resources/contributors.txt [hunk 1]
```diff
@@ -40,5 +40,5 @@ Ian321 | https://github.com/Ian321 |  | Documentation
 Yardanico | https://github.com/Yardanico |  | Documentation
 huti26 | https://github.com/huti26 |  | Documentation
 chrisduerr | https://github.com/chrisduerr |  | Documentation
-
```
Comment (zneix) [inline resources/contributors.txt:43]: Why did you remove this newline?

**Review by zneix — CHANGES_REQUESTED** [review]

File: resources/contributors.txt (hunk 1 above)
Comment (cinemassacres) [reply resources/contributors.txt:43]: was a mistake
